pylint_odoo/checkers/no_modules.py

```diff
@@ -381,8 +381,13 @@ def visit_call(self, node):
             for argument in args:
                 argument_aux = argument
                 # Check this 'name = fields.Char("name")'
-                field_name = (argument.parent.parent.targets[0].name
-                              .replace('_', ' '))
+                field_name = ''
+                if ((isinstance(argument, astroid.Const) or
+                        isinstance(argument, astroid.Keyword)) and
+                        isinstance(argument.parent.parent, astroid.Assign) and
+                        argument.parent.parent.targets):
+                    field_name = (argument.parent.parent.targets[0].name
```

Comment:
You can get the `field_name` out of the `for` because is not a argument.
`field_name = fields.Datetime(arg1, arg2='value2')`

You can get `field_name` from `call_node.parent` if is a Assign